### ISSUE TITLE
Suggest using `Array.some` instead of `Array.find(...) != null`

### ISSUE THREAD

**Issue by boris-petrov** [sindresorhus/eslint-plugin-unicorn#1258]
I saw some code in our codebase that was like: `[1, 2, 3].find(x => x === 2) != null` which could be replaced by `[1, 2, 3].some(x => x === 2)` (which then `unicorn/prefer-includes` says that could be even further simplified). It's better practice to use `some` than `find(...) != null`.

## Fail

```javascript
[1, 2, 3].find(x => x === 2) != null;
```

## Pass

```javascript
[1, 2, 3].some(x => x === 2);
```

**Comment by fisker** [comment]
We already have [`prefer-array-some`](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/prefer-array-some.md) rule, we may consider adding a check on this case.

**Comment by boris-petrov** [comment]
Ah, right, sorry, I missed that while searching for something similar. Yes, updating that rule would be nice, thanks!
